.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
As a separate issue, we should likely transition from Travis-CI to GitHub Actions.